portfolio/src/main/webapp/script.js

```diff
@@ -26,6 +26,26 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+window.addEventListener("load", function(){
+    const page = document.querySelector(".menu-page");
+    page.className += " appear";
+});
+
 function successfulSubmit() {
     alert("Thank you for contacting me");
-}
\ No newline at end of file
+}
+
+function fadeOut(){
+    var page = document.getElementsByClassName("page");
+    page[0].style.opacity = '0';
+    setTimeout(function(){window.location.href = "./main.html";}, 1000);
```

Comment:
Can we make this an arrow too? Since we use arrows below.

Reply:
I combined these into a single line with parallel style to above (.then), but I didn't use arrow functions

Reply:
i find the .then one-liner harder to read now.

I was just suggesting `setTimeout(() => { window.location.href = ... })`

Reply:
Changed to your suggestion